src/mono/sample/Android/AndroidSampleApp.csproj

```diff
@@ -15,6 +15,9 @@
     <EnableDefaultAssembliesToBundle>true</EnableDefaultAssembliesToBundle>
     <!-- With Mono AOT on Android we default to not using AOT data file optimization as it can degrade runtime performance for small binary size improvements. -->
     <_UseAotDataFile Condition="'$(RunAOTCompilation)' == 'true'">false</_UseAotDataFile>
+    <RepoDotnet Condition="'$(DOTNET_ROOT)' != ''">$(DOTNET_ROOT)/dotnet</RepoDotnet>
```

Comment:
I don't think you need this, you can just always call dotnet.cmd/dotnet.sh right?